run-dev.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cp -f ~/work/lndr/lndr-backend/data/lndr-server.dev.config ~/work/lndr/lndr-backend/data/lndr-server.config
```

Comment:
If including this file, refrain from using environment-specific absolute paths. Instead:

```
cp -f lndr-backend/data/lndr-server.dev.config lndr-backend/data/lndr-server.config
```

Reply:
This is a mistake, I'll remove it